algebird-core/src/main/scala/com/twitter/algebird/DecayedValue.scala

```diff
@@ -73,4 +73,19 @@ case class DecayedValue(value: Double, scaledTime: Double) extends Ordered[Decay
     val normalization = halfLife / math.log(2)
     value / normalization
   }
+
+  /*
+  * Moving average assuming the signal started at zero a fixed point in the past.
+  * This normalizes by the integral of exp(-t(ln(2))/halflife) from 0 to (endTime - startTime).
+  */
+  def averageFrom(halfLife: Double, startTime: Double, endTime: Double) = {
```

Comment:
working on tests, but as I'm doing them I'm becoming suspicious of the original math. For example: the average (whether using `average` or `averageFrom`) of, say, `f(t) = t` is > t, and it diverges as `t` increases. That seems like a strange thing for an average to do.